From 855763aa3ca71cb2c6ff8939417f73b928127865 Mon Sep 17 00:00:00 2001
From: Meng Dongyang <daniel.meng@rock-chips.com>
Date: Thu, 29 Mar 2018 18:28:20 +0800
Subject: [PATCH] arm64: dts: rockchip: enable u2phy low power mode for
 rk3326-863

Add rockchip,low-power-suspend property to DT of u2phy to make
the port enter lower power state when suspend.

Change-Id: I88eef53b24c8859ebe2c2b3cb5ecbdc42b406d04
Signed-off-by: Meng Dongyang <daniel.meng@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
index 7e6dca76747c..423247f633b8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
@@ -657,10 +657,12 @@
 	status = "okay";
 
 	u2phy_host: host-port {
+		rockchip,low-power-mode;
 		status = "okay";
 	};
 
 	u2phy_otg: otg-port {
+		rockchip,low-power-mode;
 		status = "okay";
 	};
 };
-- 
2.35.3

